lib/happy_hood/slack/client.rb

```diff
@@ -29,11 +29,23 @@ def self.daily_message
 
         def self.summarize_differences(differences)
           differences.reject { |d| d.valuation_difference.zero? }.map do |d|
-            "```"\
-            "(#{d.name}) - #{d.house_count} Happy Houses - (#{d.valuation_date})\n"\
-            "Yesterday: #{currency_format(d.yesterdays_valuation)}\n"\
-            "Today:     #{currency_format(d.todays_valuation)} (Difference: #{currency_format(d.valuation_difference)})\n"\
-            "```"
+            average_house_difference = if d.house_count > 1
+              avg_diff = d.valuation_difference / d.house_count
+
+              avg_diff_string = avg_diff.positive? ? "+#{currency_format(avg_diff)}" : "#{currency_format(avg_diff)}"
+
+              "(#{avg_diff_string} avg/house)"
+            end
+
+            <<~SUMMARY.strip
+              ```
+              (#{d.name}) - #{d.house_count} Happy Houses
+
+              Yesterday: #{currency_format(d.yesterdays_valuation)}
+              Today:     #{currency_format(d.todays_valuation)}
+              Difference: #{currency_format(d.valuation_difference)} #{average_house_difference}
+              ```
```

Comment:
Examples:

```
Jackson Square Townhomes (42 Happy Houses)
Yesterday:  $14,379,098.00
Today:      $14,401,431.00
Difference: $22,333.00 (+$531.74 avg/house)
```

```
Heritage Place (1 Happy House)
Yesterday:  $337,156.00
Today:      $336,920.00
Difference: -$236.00
```

```
Lake Thomas Preserve (118 Happy Houses)
Yesterday:  $58,599,827.00
Today:      $58,734,122.00
Difference: $134,295.00 (+$1,138.09 avg/house)

Reply:
Things to note:

- Difference is now on a new line
- If a neighborhood only has one house, we say `1 Happy House` not `1 Happy Houses`
- If a neighborhood only has one house, we do not include the average diff/house

Reply:
- Love the pluralization
- Difference being on a new line is MUCH clearer, nice
- Makes sense not to average when we only have one house